src/Service.Tests/dab-config.MsSql.json

```diff
@@ -2844,6 +2844,40 @@
         }
       }
     },
+    "DefaultBuiltInFunction": {
+      "source": {
+        "object": "default_with_function_table",
+        "type": "table"
+      },
+      "graphql": {
+        "enabled": true
+      },
+      "rest": {
+        "enabled": true
+      },
+      "permissions": [
+        {
+          "role": "anonymous",
+          "actions": [
+            {
+              "action": "create",
+              "fields": {
+                  "include": [
+                      "*"
+                  ],
+                  "exclude": [
+                      "current_date",
+                      "next_day"
```

Comment:
this config seems correct, but all the `mssql-commands.txt` files use `next_date`

Reply:
updated